NEWS

```diff
@@ -48,6 +48,12 @@
 		- Removing redundant and fixing implicit imports [Thoralf Klein]
 		- Hide many methods from SWIG, reducing compile memory by 500MiB [Heiko Strathmann, Fernando Iglesias, Thoralf Klein]
 
+2014-12-16 Fernando Iglesias <fernando.iglesias@shogun-toolbox.org>
```

Comment:
I will move this into Cleanups in the section above. I am wondering though, how do keep track of news constantly if we put everything in a section after a release?